tritium-lib/src/main/java/com/palantir/tritium/proxy/Instrumentation.java

```diff
@@ -39,6 +44,10 @@
  */
 public final class Instrumentation {
 
+    // TODO (davids): verify naming
+    public static final String METRIC_NAME_SERVICE = "service.response";
+    public static final String TAG_SERVICE_NAME = "service-name";
```

Comment:
wonder if we should stick the standard tag names in something like `SlsTags` or `TagNames`?

Reply:
ideally we'd have a tag-name-builder of sorts?

Reply:
yeah. can throw that together